### PR TITLE
Now ticket can be open without linked to a problem

### PR DESCRIPTION
**Pull request by leduythuccs**
Currently, the ticket model is stick with a problem, so users can't open a ticket for general issues of the website.

I add a new model (`GeneralIssue`) to resolve that problem. Now tickets can open independently from problems. 

There are still some things that I want to discuss: 
- [x] The name of the new model: GeneralIssue. Should I change it, and what should it be
- [x] The name of the new view: NewTicketIssueView. Should I change it?
- [x] The URL to open new issue: Where should it be? I place it in `/tickets/new`
- [x] Should the URL to the issue is required while filling the ticket form? 

Preview: 
- Issue Form:
image

- Homepage: 
image

- Ticket view: 
image

The UI of opening a ticket for the problem is still the same

### CODE REVIEW

**Comment by hung3a8** [comment]
Are there any easier ways to access the GeneralIssue Form than typing the URL everytime?
Also, I think we should remove the old ticket form that directly links to the Problems. We can replace it with the URL to the GeneralIssue form, with the `issue_url` param is the problem links.

File: judge/views/ticket.py [hunk 1]
```diff
@@ -76,6 +84,40 @@ def form_valid(self, form):
...
+    template_name = 'ticket/new_issue.html'
+
+    def get_form_kwargs(self):
+        kwargs = super(NewIssueTicketView, self).get_form_kwargs()
+        kwargs['request'] = self.request
+        kwargs['issue_url'] = self.request.GET.get('issue_url', '')
+        return kwargs
+
+    def get_title(self):
+        print("gettitil")
```
Comment (hung3a8) [inline judge/views/ticket.py:98]: Is this debug print :)?

Comment (leduythuccs) [reply judge/views/ticket.py:98]: = )))) yep
lmao

**Comment by leduythuccs** [comment]
> Are there any easier ways to access the GeneralIssue Form than typing the URL everytime?

Add a button to the navbar I guess.

Anw, I don't think we should remove the old ticket models. Because many feature like 
image
image

will be much more complicated to implement and it not worth it to re-implement.

File: templates/base.html [hunk 2]
```diff
@@ -233,10 +233,15 @@
         </ul>
 
         <span id="user-links">
-            <a id="flag_vi" href="#" onclick="submmit_language('vi')" style="margin-right: 3px;">
+            <a href="/tickets/new" style="margin-right: 5px;">
```
Comment (leduythuccs) [inline templates/base.html:236]: can we make it to /tickets/news?issue_url=`current url`. 

That will make user don't have to type the url of the current page.

Comment (hung3a8) [reply templates/base.html:236]: Good idea =))
Can u do that right now?

**Comment by leduythuccs** [comment]
The report issue form should require user login

**Review by hung3a8 — APPROVED** [review]
lgtm =))

**Pull request merged**